alibi_detect/cd/pytorch/tests/test_mmd_online_pt.py

```diff
@@ -22,12 +22,20 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return self.dense2(x)
 
 
+def preprocess_list(x: List[np.ndarray]) -> np.ndarray:
+    if len(x) > 1:
+        return np.concatenate(x, axis=0)
+    else:
+        return np.array(x)[0]
```

Comment:
See above. Can we factor this utility function out in some test utility file? Or even library utility if there is a use case for using it?